x/evm/keeper/msg_server.go

```diff
@@ -1223,6 +1224,8 @@ func (s msgServer) CreatePendingTransfers(c context.Context, req *types.CreatePe
 		return nil, err
 	}
 
+	keeper := s.ForChain(chain.Name)
```

Comment:
Why?

Reply:
Trying to keep the `chain.Name` usage consistent. Some places used `req.Chain` instead of `chain.Name`. For logging purposes and events they can differ in case.